indico/modules/events/tracks/models/groups.py

```diff
@@ -0,0 +1,71 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2019 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from indico.core.db.sqlalchemy import db
+from indico.core.db.sqlalchemy.descriptions import DescriptionMixin, RenderMode
+from indico.modules.events.tracks.models.tracks import get_next_position
+from indico.util.locators import locator_property
+from indico.util.string import format_repr, return_ascii, text_to_repr
+
+
+class TrackGroup(DescriptionMixin, db.Model):
+    __tablename__ = 'track_groups'
+    __table_args__ = {'schema': 'events'}
+
+    possible_render_modes = {RenderMode.markdown}
+    default_render_mode = RenderMode.markdown
+
+    id = db.Column(
+        db.Integer,
+        primary_key=True
+    )
+
+    title = db.Column(
+        db.String,
+        nullable=False
+    )
+
+    position = db.Column(
+        db.Integer,
+        nullable=False,
+        default=get_next_position
+    )
+
+    event_id = db.Column(
+        db.Integer,
+        db.ForeignKey('events.events.id'),
+        index=True,
+        nullable=False
+    )
+
+    event = db.relationship(
+        'Event',
+        lazy=True,
+        backref=db.backref(
+            'track_groups',
+            cascade='all, delete-orphan',
+            lazy=True,
+            order_by=id
+        )
+    )
+
+    # relationship backrefs:
+    # - tracks (Track.track_group)
+
+    @property
+    def is_track_group(self):
```

Comment:
that shouldn't be a property but just a simple attribute: `is_track_group = True` (near the top of the class, we did similar things in other models like User)